addons/website_sale/static/src/css/website_sale.css

```diff
@@ -438,3 +438,12 @@ ul.wizard li:first-child {
   padding-left: 15px;
   border-radius: 4px 0 0 4px;
 }
+
+.vertical-align {
+  display: flex;
+  align-items: center;
+}
+
+.label-text-align {
+  vertical-align: 15%;
+}
```

Comment:
You should modify the original `.sass` file and then recompile it into CSS instead of directly modifying the `.css` file.